plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -24,6 +26,7 @@ var sampleConfig string
 // SQLServer struct
 type SQLServer struct {
 	Servers      []string        `toml:"servers"`
+	Timeout      config.Duration `toml:"timeout"`
```

Comment:
Can we please name this `QueryTimeout`, same for the option to eliminate confusion
```suggestion
	QueryTimeout      config.Duration `toml:"query_timeout"`
```